next.clayui.com/content/docs/components/select.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Select"
+description: "A form control element used to select from several provided options and enter data."
+packageNpm: "@clayui/select"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {Select, SelectWithOption} from '../../../src/components/clay/Select';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
+    - [SelectWithOption](#selectwithoption)
+- [API](#api)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/forms/selector/">Selectors Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+## Variants and Best Pratices
+
+### Default
+
+`ClaySelect` and `ClaySelect.Option` are `<select>` and `<option>` elements respectivetly that are styled using ClayCSS classes.
+
+`ClaySelect` represents a control that provides a menu of options (`ClaySelect.Option`). `ClaySelect.Option` defines an option in a select list.
+
+Use `ClaySelect.Option` for wrapping an option item.
+
+<Select />
+
+### SelectWithOption
+
+If you use ClaySelect only for simple cases that do not need props for [`options`](#api-options), you can use `<ClaySelectWithOption />` which will have the same result as the previous version.
+
+<SelectWithOption />
+
+## API
+
+### Select and Select.Option Variants
```

Comment:
We can probably remove this section since it doesn't really provide any extra information that the user wouldn't already assume.